redisinsight/ui/src/components/new-index/CreateIndexStep/CreateIndexStepWrapper.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { TabsProps } from '@redis-ui/components'
+import Tabs, { TabInfo } from 'uiSrc/components/base/layout/tabs'
+import { BuildNewIndexTabTrigger } from './BuildNewIndex/BuildNewIndexTabTrigger'
+
+export enum VectorIndexTab {
+  BuildNewIndex = 'build-new-index',
+  UsePresetIndex = 'use-preset-index',
+}
+
+const VECTOR_INDEX_TABS: TabInfo<string>[] = [
+  {
+    value: VectorIndexTab.BuildNewIndex,
+    label: <BuildNewIndexTabTrigger />,
+    content: null,
+    disabled: true,
+  },
+  {
+    value: VectorIndexTab.UsePresetIndex,
+    label: 'Use preset index',
+    content: <div>TODO: Add content later</div>,
+  },
+]
+
+export const CreateIndexStepWrapper = (props: Partial<TabsProps>) => {
+  const { tabs, defaultValue, ...restProps } = props
```

Comment:
nit: I think `rest` is fine instead of `restProps` (react community practice)

Reply:
I saw it as a pattern in `redis-ui` and thought it was the convention I must follow. 
Anyway, `restProps` is updated to `rest` now ✌️